templates/security/cmmc/index.html

```diff
@@ -46,8 +46,8 @@ <h1>
         <div class="p-image-container--3-2 is-cover">
           {{ image(url="https://assets.ubuntu.com/v1/0a4cf754-hero-img.jpg",
                         alt="",
-                        width="568",
-                        height="852",
+                        width="600",
+                        height="400",
                         hi_def=True,
                         loading="lazy",
```

Comment:
Could you please set "loading" to "auto" since this image will be right on user view when page is loaded and is better if not lazy-loaded.